packages/taxdat/R/model_evaluation_helpers.R

```diff
@@ -329,19 +331,28 @@ plot_raster_covariates <- function(covar_data_filename,
         one_of(dimnames(covar_cube_output$covar_cube)[[3]]), key = "covars", value = "value"
       )
     
-    plt <- plt +
-      ggplot2::geom_sf(
-        data = pltdata_dummy,
-        ggplot2::aes(fill = value, color = value)
-      ) +
-      ggplot2::scale_fill_viridis_c(aesthetics = c("colour", "fill"),
-                                    guide = ggplot2::guide_colorbar(title = "Covariate at time 1"),
-                                    option = "B",
-                                    na.value = "white")  +
-      ggplot2::theme_bw() +
-      # ggplot2::scale_fill_continuous("Covariate at time 1") +
-      ggplot2::theme(legend.position = "bottom") +
-      ggplot2::facet_wrap(~covars)
+    # plt <- plt +
+    #   ggplot2::geom_sf(
+    #     data = pltdata_dummy,
+    #     ggplot2::aes(fill = value, color = value)
+    #   ) +
+    #   ggplot2::scale_fill_viridis_c(aesthetics = c("colour", "fill"),
+    #                                 guide = ggplot2::guide_colorbar(title = "Covariate at time 1"),
+    #                                 option = "B",
+    #                                 na.value = "white")  +
+    #   ggplot2::theme_bw() +
+    #   # ggplot2::scale_fill_continuous("Covariate at time 1") +
+    #   ggplot2::theme(legend.position = "bottom") +
+    #   ggplot2::facet_wrap(~covars)
+    plt<-pltdata_dummy%>%group_by(covars)%>%
+      do(gg={ggplot(.,ggplot2::aes(fill = value, color = value)) + ggplot2::geom_sf()+
+          ggplot2::facet_wrap(~covars)+ggplot2::scale_fill_viridis_c(aesthetics = c("colour", 
+                                                                                    "fill"), guide = ggplot2::guide_colorbar(title = "Covariate at time 1"), 
```

Comment:
This line is too long.  are we using formatr here?

Reply:
nope, will update it with formatR

Reply:
the tidy_source seems to also create long code? i updated by making new a new row for each new function?

Reply:
sorry, there's a formatR hook that should be committing when you push. It's set up with a style guide for our group.  Do you not have that? (Yeah, it will create long code, though hopefully not too wide)

Reply:
not sure if I have that? is there a way that i can check?